app/controllers/candidate_interface/sign_in_controller.rb

```diff
@@ -40,6 +34,26 @@ def create
       end
     end
 
+    def create_expired
```

Comment:
Perhaps move this down below the expired method? Not sure about the name but can't think of something else.

Reply:
I'm not sure about the name either. `create_from_expired_token`?

Reply:
`create_for_known_user`?